thorlcr/msort/tsorts.cpp

```diff
@@ -91,6 +91,7 @@ class CWriteIntercept : public CSimpleInterface
         if (idxFileStream.get())
         {
             idxFileStream->write(sizeof(o),&o);
+            idxFile->noteSize(idxFileIO->getStatistic(StSizeDiskWrite));
```

Comment:
writeidxofs is going to be called per row, this should be moved to after the idxFileStream->flush() (~ line 268)